int8 axis
---
bool success
